manim/utils/iterables.py

```diff
@@ -163,7 +196,9 @@ def listify(obj) -> list:
         return [obj]
 
 
-def make_even(iterable_1: Iterable, iterable_2: Iterable) -> tuple[list, list]:
+def make_even(
+    iterable_1: Iterable[T], iterable_2: Iterable[U]
+) -> tuple[list[T | U], list[T | U]]:
```

Comment:
```suggestion
) -> tuple[list[T], list[U]]:
```
According to the implementation. Am I right?

Reply:
Believe so!